include/wx/gtk/textctrl.h

```diff
@@ -10,6 +10,11 @@
 #ifndef _WX_GTK_TEXTCTRL_H_
 #define _WX_GTK_TEXTCTRL_H_
 
+#include <gtk/gtk.h>
```

Comment:
We shouldn't include GTK+ headers from our own public headers. This means, of course, that we can't use `GTK_CHECK_VERSION` in them neither, but this is not really a problem, we can just use `__WXGTK3__` as the guard here and forward declare `GtkEventController`.